cypress/pages/workflow.ts

```diff
@@ -3,9 +3,46 @@ import { BasePage } from "./base";
 export class WorkflowPage extends BasePage {
 	url = '/workflow/new';
 	getters = {
-		workflowNameInput: () => cy.getByTestId('workflow-name-input', { timeout: 5000 }).then($el => cy.wrap($el.find('input'))),
+		// workflowNameInput: () => cy.getByTestId('workflow-name-input', { timeout: 5000 }).then($el => cy.wrap($el.find('input'))),
 		workflowImportInput: () => cy.getByTestId('workflow-import-input'),
 		workflowTags: () => cy.getByTestId('workflow-tags'),
-		saveButton: () => cy.getByTestId('save-button'),
+		saveButton: () => cy.getByTestId('wf-save-button').children().first(),
+		activatorSwitch: () => cy.getByTestId('wf-activate-switch'),
```

Comment:
Nitpick — we should be consistent in naming. Here and above, we use abbreviated `wf-*` but the rest of the ids are `workflow-*`

Reply:
Agree.